chia/types/coin_spend.py

```diff
@@ -39,7 +40,11 @@ def hints(self) -> List[bytes]:
         generator = simple_solution_generator(bundle)
 
         npc_result = get_name_puzzle_conditions(
-            generator, INFINITE_COST, cost_per_byte=DEFAULT_CONSTANTS.COST_PER_BYTE, mempool_mode=False
+            generator,
+            INFINITE_COST,
+            cost_per_byte=DEFAULT_CONSTANTS.COST_PER_BYTE,
+            mempool_mode=False,
+            height=uint32(2000000),
```

Comment:
the `hints()` function is only called from the wallet, on our own coins, so this should be safe. I have another outstanding PR that moves this function into the wallet to make this clear